css/code.scss

```diff
@@ -24,9 +24,10 @@ code.highlighter-rouge {
 div.highlighter-rouge {
   margin: 40px 0;
   @include trim-v-margins;
-  background: $gray-500;
+  background: #2d2d2d;
   overflow-x: auto;
   overflow-y: auto;
+  text-align:left;
```

Comment:
Nice! These changes both make sense, but I wonder if `code.highlighter-rouge`'s background (a few lines above) should also be changed to be this div's color. It could also be vice versa, as I find the `code.highlighter-rouge`'s background sufficiently dark to read it.

Reply:
While we're tweaking the appearance of code blocks, it might be nice to include a bit of rounding on the boxes, e.g. `border-radius: 3px`. I don't know how that plays into the visual language of the site, so I'll leave it to @vincerubinetti to approve that or not.

Reply:
I will add a $rounded variable to the upstream lab website template for users who want that.

Reply:
Thank you @vincerubinetti and @falquaddoomi !